src/main/java/ui/components/pickers/LabelPicker.java

```diff
@@ -49,6 +50,9 @@ private void showLabelPicker(TurboIssue issue) {
     private void replaceLabels(TurboIssue issue, List<String> labels) {
         List<String> originalLabels = issue.getLabels().stream().sorted().collect(Collectors.toList());
         if (!labels.equals(originalLabels)) {
+            List<Action> actions = getActions(originalLabels, labels);
+            List<String> resultingLabels = getResultingLabels(originalLabels, actions);
```

Comment:
These methods can be named better. This is sort of like computing a diff and applying a diff between sets of labels, right?

Also, does this part do anything? It seems that we still end up with `labels` as the result, and we don't use `actions`.